### ISSUE TITLE
Is this repo still alive?

### ISSUE THREAD

**Issue by vitcpp** [pgsphere/pgsphere#7]
Dear @mnullmei, is this repo still alive? I have some questions about this repo. How can I contact to you?

**Comment by df7cb** [comment]
Hi @vitcpp, it looks like I'm the only person left in the pgsphere organization. It was originally created as a new home for pgsphere since the https://github.com/postgrespro/pgsphere repo didn't look active anymore.

If the https://github.com/postgrespro/pgsphere repo is alive (again), and accepts contributions from the astronomy community, we can shut this one down.

**Comment by vitcpp** [comment]
Hi @df7cb, thank you very much for the response! I'm happy to say that postgrespro/pgsphere is alive now. We are working on polishing the repo now. We are ready to work with the community on project improvements and accept new PRs. I'm assigned to support pgsphere in the PostgresPro organization, and I will do my best for the community. I graduated as an astronomer, but for the last two decades I have worked as a software developer. I'm familiar with the astronomical domain. I would be happy to work on pgsphere improvements. To make things easier, I propose to use postgrespro/pgsphere as the primary place for pgsphere development.
